### ISSUE TITLE
Exception when Context contains [Keyless] Views

### ISSUE THREAD

**Issue by W0lfgang** [KorzhCom/EasyData#62]
DbContext implementation:
```
protected override void OnModelCreating(ModelBuilder modelBuilder)
{
    modelBuilder.Entity<MyCustomView>(entity =>
    {
        entity.ToView("MyCustomView", "dbo");
    });
    // ...
}
```

Model:
```
[Keyless]
public partial class MyCustomView
{
    // ...
}
```

Snippet from my dev tools:

```
GET https://localhost:44346/api/easydata/models/__default returned HTTP Status Code 400: Bad Request
easydata.min.js:2 Error: Value cannot be null. (Parameter 'key'). Source: undefined
```

Sure, I might add an `[MetaEntity(false)]` to my view's POCO representation, but lot of my contexts have lots of views. It should be possible to detect and just skip `[Keyless]` entities.

**Comment by antifree** [comment]
Yep. That's a good point. We will add such fix

**Comment by korzh** [comment]
We have fixed this in 1.3.6-rc01. The final release will be available in a few days.

**Comment by antifree** [comment]
Final version 1.3.6 with the fix has been released!
